### DISCUSSION TITLE
만다라트와 함께하는 2024년 계획 - 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#58] [Announcements]
# 만다라트와 함께하는 2024년 계획 - 정현수 기술 블로그

2024년 계획을 세워보자.

https://junghyeonsu.com/posts/2024-plan-with-mandalart/

**Comment by wherehows** [comment]
멋있네용! 영감 받고가용 👍👍👍👍👍👍

**Reply by junghyeonsu** [reply]
감사합니다!

**Comment by haryan248** [comment]
화이팅입니다! 😃😃

**Comment by JoonheeJeong** [comment]
현수야~ 오랜만에 현수 블로그 생각나서 들렀어 ㅎㅎ 

꾸준히 성장하는 현수 보기 좋다! 덕분에 나도 도전 받고 간다! 

작년 이맘때 현수 번아웃 글을 봤었던 것 같은데, 건강하고 활기차게 잘 지내길 빈다!! 그리고 나도 그래야겠다 ㅎㅎ

**Reply by junghyeonsu** [reply]
형 오랜만이네 저번에 볼수있었는데 아쉽ㅋㅋㅜㅜ
다음에 대전에서 꼭 보자~~!
